app/models/work.rb

```diff
@@ -270,7 +270,7 @@ def update_tag_index
   end
 
   def self.work_blurb_tag_cache_key(id)
-    "/v1/work_blurb_tag_cache_key/#{id}"
+    "/v2/work_blurb_tag_cache_key/#{id}"
```

Comment:
Should we be expiring the view for the blurb and meta, too? Or will this force them to update? 

https://github.com/otwcode/otwarchive/blob/bd4f323003a7f63d0e708ff334fd666e4760601a/app/views/works/_work_module.html.erb#L27

https://github.com/otwcode/otwarchive/blob/bd4f323003a7f63d0e708ff334fd666e4760601a/app/views/works/_meta.html.erb#L1

Reply:
Those will be updated, as they use `Work.work_blurb_tag_cache`:

https://github.com/otwcode/otwarchive/blob/0ec885aac78cfa16027d0e672649300c34b55e06/app/models/work.rb#L276-L278

which itself is a cached value, whose key we just changed.

If we need to expire just the blurbs, or just the metas, but not both, then we would change the keys in the views.

Reply:
If I had actually _read_ the lines in the views beyond the word "cache", I would have seen that. 🤦‍♂